src/services/Functional/User.js

```diff
@@ -11,21 +11,25 @@ export default {
         store.state.auth.reAuthenticationPromise == null ||
         UtilitiesService.isObjectEmpty(store.state.auth.reAuthenticationPromise)
       ) {
-        store.dispatch("auth/unsetAccessToken");
-        store.dispatch("auth/setReAuthenticationState", false);
-        return Promise.resolve(true);
+        store.dispatch("auth/autoLogoutUser");
+        return Promise.reject("Auto Logout!");
```

Comment:
I was still getting the red exception for "Auto Logout". Resolved it by using this instead. Read [this](https://stackoverflow.com/questions/44678505/uncaught-in-promise) to understand more. We were sending the rejected promise instead of the promise!

```python
return new Promise((reject) => {
          reject("Auto Logout!")
        });
```

To replicate, I made this change in `src/App.vue`:
<img width="783" alt="Screenshot 2021-11-01 at 11 50 46 AM" src="https://user-images.githubusercontent.com/13185699/139630175-3dca6aa9-23dd-44c4-8649-682371fc11fe.png">
<img width="644" alt="Screenshot 2021-11-01 at 11 50 52 AM" src="https://user-images.githubusercontent.com/13185699/139630193-32cd33f9-0bbf-458d-ae2e-7c7fe104cf4a.png">